src/lib/__mocks__/geminiUtils.ts

```
@@ -0,0 +1,16 @@
+// src/lib/__mocks__/geminiUtils.ts
+import { vi } from 'vitest';
+
+export const generateStoryFromWords = vi.fn();
+
+export class GeminiGenerationError extends Error {
+  details: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)